README.md

```diff
@@ -3,7 +3,11 @@ sftp
 
 The `sftp` package provides support for file system operations on remote ssh servers using the SFTP subsystem.
 
-[![UNIX Build Status](https://travis-ci.org/pkg/sftp.svg?branch=master)](https://travis-ci.org/pkg/sftp) [![GoDoc](http://godoc.org/github.com/pkg/sftp?status.svg)](http://godoc.org/github.com/pkg/sftp)
+This is a fork of [github.com/pkg/sftp](http://github.com/pkg/sftp) that changes the server to allow for a plugable backend and adds an
+S3 and file system backend. The file system backend is mostly used for tests and is similar to the previous behavior. Additionally, this
+adds a ManagedServer component that can be used to easily create a server with an arbitrary backend.
+
```

Comment:
That's funny, are you aware that pkg/sftp already has a pluggable backend? Please read the docs about the [request-server](https://pkg.go.dev/github.com/pkg/sftp#RequestServer). For example SFTPGo implements all the features that I see here and much more using the request server.

Obviously you are free to fork the project and make the changes you like :smile:

Reply:
Thanks for the info here! For more context, this repo was forked roughly 9 years ago and is part of a legacy system that was created before these other options existed. This PR was a misstep on my part as I just meant to push a minor update to our fork. But thanks for the info on request-server, definitely worth looking into for us.